src/main/java/seedu/address/logic/commands/ListTutorCommand.java

```diff
@@ -9,9 +9,9 @@
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListTutorCommand extends Command {
+public class ListTutorCommand extends ListCommand {
 
-    public static final String COMMAND_WORD = "list_t";
+    //public static final String COMMAND_WORD = "list_t";
 
     // the full string that calls list tutor which is used by UI
     // please update after changing list_t to list tutor/list t
```

Comment:
Similar to ListStudentCommand, will need to change line19 to public static final String COMMAND_LIST_TUTOR_STRING = "list tutor";